pkg/driver/node.go

```diff
@@ -165,7 +165,7 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				}
 			}
 
-			if f == "awscredsuri" {
+			if strings.HasPrefix(f, "awscredsuri") {
 				klog.Warning("awscredsuri mount option is not supported by efs-csi-driver.")
 				return nil, nil
```

Comment:
i know this was already here but it doesnt look right, return nil nil? no response, no error? should it return nil ,err?

Reply:
right, we have been debating on putting nil or error here, we then realized this might break current user workflow if they are already using this mount option so chose to return nil than error.

Reply:
Also, return nil nil causes FailedMount.

```
Warning  FailedMount      14s                 kubelet            Unable to attach or mount volumes: unmounted volumes=[persistent-storage], unattached volumes=[kube-api-access-9c7nq persistent-storage]: timed out waiting for the condition
  Warning  FailedMount      9s (x9 over 2m17s)  kubelet            MountVolume.SetUp failed for volume "efs-pv" : rpc error: code = Internal desc = grpc: error while marshaling: proto: Marshal called with nil
```